### PR TITLE
About intensities (gamma correction, int/float)

### PR DESCRIPTION
**Pull request by MikhailRyazanov**
The proposed addition to README from [here](https://github.com/PyAbel/PyAbel/issues/303#issuecomment-698072238).
Please check whether the wording can be improved or something added/removed...

### CODE REVIEW

**Review by stggh — APPROVED** [review]
Looks good to me. Thanks, for "exposing" the subtle gamma intensity correction.

File: README.rst [hunk 1]
```diff
@@ -147,17 +147,20 @@ The PyAbel code adheres to the following conventions:
...
 
+-
+    **Intensity:** Absolute pixel intensities can be arbitrary (withing the floating-point range), but the intensity scale must be linear. Keep in mind that cameras and common image formats can use `gamma correction <https://en.wikipedia.org/wiki/Gamma_correction>`_ and thus provide data with nonlinear intensity encoding. When working with such data, make sure to apply appropriate intensity transformations before the analysis. Most PyAbel methods also assume intensities to be floating-point numbers, and when applied to integer types, can return inappropriately rounded results. The ``abel.Transform`` class recasts the input image to ``float64`` by default, but if you wish to call the transform methods directly or use other tools, you might need to perform the conversion yourself (as ``IM.astype(float)``, for example).
```
Comment (DanHickstein) [inline README.rst:162]: Bold indicates suggested changes:

"**The pixel intensities can have any value (within the floating-point range). However, the intensity scale must be linear.**"

"...cameras and common image formats **often** use" 

"...nonlinear intensity encoding. **Thus, if possible, it is recommended to disable the gamma correction on cameras used to record images that will be inverse Abel transformed. If this is not possible, then it is necessary to apply the appropriate intensity transformations before the analysis.**"

Also, what rounding issues arise with the int-to-float conversion?

**Review by DanHickstein — APPROVED** [review]
Looks good to me! I made some minor suggestions. Take or leave them.

**Comment by DanHickstein** [comment]
>  Thanks, for "exposing" the subtle gamma intensity correction.

Hahahaha!

File: README.rst (hunk 1 above)
Comment (MikhailRyazanov) [reply README.rst]: Thanks!

**Comment by MikhailRyazanov** [comment]
> Also, what rounding issues arise with the int-to-float conversion?

They sometimes arise _without_ this conversion.

I've noticed when working on #302 that, for example, `scipy.ndimage.shift()` with a fractional shift applied to an integer array returns also an integer array, apparently rounded:
```python
import numpy as np
from scipy.ndimage import shift

a = np.array([0, 1, 2, 1, 0])

print(shift(a, 0.5))
# [0 0 2 2 0]

print(shift(a.astype(float), 0.5))
# [0.     0.3125 1.6875 1.6875 0.3125]
```
which is probably not very useful. I haven't checked other functions, but there might be similar surprizes. And, well, the `abel.Transform()` docstring says: “Many images are imported in other formats (such as `uint8` or `uint16`), and this does not always play well with the transorm algorithms.”

**Comment by DanHickstein** [comment]
Okay, I see how this could be an issue. In reality, the issue probably only manifests when the integers are low value. But, it's good that abel.Transform converts them to float. Potentially the individual transform functions (and other functions) should cast them to floats as well. But this would be a lot of work.
Anyway, I think that you can merge this.

**Pull request merged**